_UI/_web_interface/tooltips.py

```diff
@@ -132,10 +132,10 @@
         ),
         dbc.Tooltip(
             [
-                html.P("VFO Noise Timeout allows to disable writing signal on disk if it exceeded some timeout"),
-                html.P("-1 means no timeout is applied"),
+                html.P("Maximum Demod Time allows to disable writing signal on disk if it exceeded some timeout"),
+                html.P("* < 0 or 0 means no timeout is applied"),
```

Comment:
Please see above, if min value of 0 is enforced on input, then no need to mention negative numbers.

Reply:
Done